metadata-editor/app/controllers/EditsController.scala

```diff
@@ -48,6 +48,7 @@ object EditsController extends Controller with ArgoHelpers {
 
   val Authenticated = EditsApi.Authenticated
   val dynamo = new DynamoDB(Config.awsCredentials, Config.dynamoRegion, Config.editsTable)
+  def couldNotPublishMessage(id: String) = respondError(InternalServerError, "message-not-published", s"Data saved, but indexing could not take place")
```

Comment:
For precision, "notification/message could not be published/sent"? (this component just sends a message, what is done with it is not its concern)